src/Perl6/Metamodel/EnumHOW.nqp

```diff
@@ -85,13 +88,18 @@ class Perl6::Metamodel::EnumHOW
     }
 
     method enum_from_value($obj, $value) {
-        unless %!value_to_enum {
+        my $value_to_enum := $!value_to_enum;
+        unless $value_to_enum {
+            $value_to_enum := nqp::hash;
```

Comment:
Hm, I thought in NQP it had to be `nqp::hash()`, but apparently it somehow works anyway.